### PR TITLE
upgrade invoker docker client to 1.12.0

### PR DESCRIPTION
**Pull request by domdom82**

### CODE REVIEW

**Comment by domdom82** [comment]
PG1 730

File: core/invoker/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 FROM scala
 
 # Uncomment to fetch latest version of docker instead: RUN wget -qO- https://get.docker.com | sh
-# Install docker 1.9.0
-RUN wget --no-verbose -O /usr/bin/docker "https://get.docker.com/builds/Linux/x86_64/docker-1.9.0" && \
+# Install docker 1.12.0 Client
+RUN wget --no-verbose https://get.docker.com/builds/Linux/x86_64/docker-1.12.0.tgz && \
+tar --strip-components 1 -xvzf docker-1.12.0.tgz -C /usr/bin docker/docker && \
+rm -f docker-1.12.0.tgz && \
```
Comment (markusthoemmes) [inline core/invoker/Dockerfile]: Is there a chance that these stay installable like that? So we could have an ENV variable in here

`DOCKER_VERSION=1.12.0` and use it throughout the URLs for an easier upgrade in the future? What do you think?

Comment (domdom82) [inline core/invoker/Dockerfile]: Well I suppose so. They just changed from single file to tar.gz but that was due to the split into multiple binaries. I think it should at least stay as tar.gz for the next couple of releases. I will update the PR accordingly.

**Comment by markusthoemmes** [comment]
LGTM. Leaving for @perryibm for final verification.

**Comment by perryibm** [comment]
PG2-454 had a ton of failures.  Re-running as PG2-456.

**Comment by domdom82** [comment]
finally got a green PG2-470

**Comment by markusthoemmes** [comment]
rebased and merged in 9c6e7d2f77909cf84aa8a0c3f3189791af76b3a0

**Pull request closed without merging**